docs/src/main/asciidoc/security-overview-concept.adoc

```diff
@@ -2,264 +2,33 @@
 = Quarkus Security overview
 include::_attributes.adoc[]
 :categories: security
-
+ 
 Quarkus Security is a framework that provides the architecture, multiple authentication and authorization mechanisms, and other tools for you to build secure and production-quality Java applications.
 
 == Getting started with Quarkus Security
 
-Before you start building security into your Quarkus applications, review the overview information to learn about the Quarkus Security architecture and the different authentication and authorization mechanisms that Quarkus supports.
+Before you start building security into your Quarkus applications, review the following resources to help you understand the security architecture and what's supported.
+
+* xref:security-architecture-concept.adoc[Quarkus Security architecture]
+* xref:security-authentication-mechanisms-concept.adoc[Supported authentication mechanisms in Quarkus]
 
 To get started with security in Quarkus, we recommend that you first combine the Quarkus built-in xref:security-basic-authentication-concept.adoc[Basic HTTP authentication] with the JPA identity provider to enable role-based access control (RBAC). 
 Complete the steps in the xref:security-basic-authentication-tutorial.adoc[Secure a Quarkus application with Basic authentication] tutorial.
 After you have successfully secured your Quarkus application with basic HTTP authentication, you can increase the security further by adding more advanced authentication mechanisms, for example, OpenID Connect (OIDC) authentication.
 
-== Security architecture
```

Comment:
Note: All of this H2 section of security-overview-concept.adoc is now moved as-is into a new concept topic in this PR named security-architecture-concept.adoc.

Reply:
@michelle-purcell Good idea, having an architecture diagram in the dedicated concept doc would work well as well, I'll ping you later after the PR is done to discuss